src/components/RadioButton/RadioButton.vue

```diff
@@ -30,8 +30,9 @@
         { 'cursor-not-allowed': disabled || readonly }
       ]"
     >
-      {{ label }}
-      <slot />
+      <slot>
```

Comment:
When we have a slot like this, does the slot content get rendered above or below the label?

Reply:
Uhh, neither. 😬  The label will just be the default slot content if nothing is provided. Otherwise if you provide something, you can put whatever you want in there in whatever order you'd like. ☺️ (Also, will update docs to reflect this.)